dist/css/_variables.css

```diff
@@ -61,162 +62,36 @@
-  --font-body-3-bold-text-decoration: none;
-  --font-body-3-bold-paragraph-spacing: 0;
-  --font-body-3-bold-letter-spacing: 0.005em;
-  --font-body-3-bold-font-size: 0.75rem;
-  --font-body-3-bold-line-height: 16px;
-  --font-body-3-bold-font-weight: 700;
-  --font-body-3-bold-font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;;
-  --font-body-3-text-case: none;
-  --font-body-3-text-decoration: none;
-  --font-body-3-paragraph-spacing: 0;
-  --font-body-3-letter-spacing: 0.005em;
-  --font-body-3-font-size: 0.75rem;
-  --font-body-3-line-height: 16px;
-  --font-body-3-font-weight: 400;
-  --font-body-3-font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;;
-  --font-body-2-bold-text-case: none;
-  --font-body-2-bold-text-decoration: none;
-  --font-body-2-bold-paragraph-spacing: 0;
-  --font-body-2-bold-letter-spacing: 0.005em;
-  --font-body-2-bold-font-size: 0.875rem;
-  --font-body-2-bold-line-height: 20px;
-  --font-body-2-bold-font-weight: 700;
-  --font-body-2-bold-font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;;
-  --font-body-2-text-case: none;
-  --font-body-2-text-decoration: none;
-  --font-body-2-paragraph-spacing: 0;
-  --font-body-2-letter-spacing: 0.005em;
-  --font-body-2-font-size: 0.875rem;
-  --font-body-2-line-height: 20px;
-  --font-body-2-font-weight: 400;
-  --font-body-2-font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;;
-  --font-body-1-bold-text-case: none;
-  --font-body-1-bold-text-decoration: none;
-  --font-body-1-bold-paragraph-spacing: 0;
-  --font-body-1-bold-letter-spacing: 0.005em;
-  --font-body-1-bold-font-size: 1rem;
-  --font-body-1-bold-line-height: 24px;
-  --font-body-1-bold-font-weight: 700;
-  --font-body-1-bold-font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;;
-  --font-body-1-text-case: none;
-  --font-body-1-text-decoration: none;
-  --font-body-1-paragraph-spacing: 0;
-  --font-body-1-letter-spacing: 0.005em;
-  --font-body-1-font-size: 1rem;
-  --font-body-1-line-height: 24px;
-  --font-body-1-font-weight: 400;
-  --font-body-1-font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;;
-  --line-height-4xl: 70px;
-  --line-height-3xl: 56px;
-  --line-height-2xl: 40px;
-  --line-height-xl: 32px;
-  --line-height-lg: 28px;
-  --line-height-base: 24px;
-  --line-height-sm: 20px;
-  --line-height-xs: 16px;
-  --line-height-2xs: 14px;
+  --font-monospace-1: 500 1.75rem/2rem 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
```

Comment:
My issues here is that sub, sup, and small children will not adjust line height correctly. However, if this has been tested, and the experience feels right, then I would think this was good.

Reply:
Would the calc be better? 

```
 --font-control-body-1: 400 1rem/calc(20 / 16) 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
 ```

To really solve this, we need to make a change at the system level which is a separate scope. I mainly want to get rid of the individual typography tokens with this PR. But if this subtracts from the experience, I'm also totally okay with closing this and leaving as is